lib/__tests__/ProjectLogsManager.ts

```diff
@@ -1,14 +1,14 @@
 // @ts-nocheck
```

Comment:
This file is missing the `.test` in the file name, would you mind updating it while we are in here? 🙏

Reply:
ah good catch, done in [dd75ef5](https://github.com/HubSpot/hubspot-cli/pull/1219/commits/dd75ef54f0f87d557689dc2620071c3bc8ae6172)